.github/workflows/smoke.yml

```diff
@@ -56,8 +56,15 @@ jobs:
 
           # Set the step output
           echo "suites=$(cat suites.json)" >> $GITHUB_OUTPUT
+  download-cli:
+    uses: dependabot/smoke-tests/.github/workflows/download-cli.yml@main
```

Comment:
Noteworthy: I asked it to reuse the reusable Download CLI workflow from the smoke test repo. There might be more reuse in this workflow, but thought this would be nice.